src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts

```diff
@@ -139,12 +113,15 @@ export class InstructorSessionResultPageComponent implements OnInit {
               private courseService: CourseService,
               private studentService: StudentService,
               private instructorService: InstructorService,
-              private commentService: FeedbackResponseCommentService,
               private route: ActivatedRoute,
-      private timezoneService: TimezoneService, private statusMessageService: StatusMessageService,
-      private modalService: NgbModal, private router: Router,
-      private commentsToCommentTableModel: CommentsToCommentTableModelPipe,
-      private commentToCommentRowModel: CommentToCommentRowModelPipe) {
+              private timezoneService: TimezoneService,
+              private modalService: NgbModal,
+              private router: Router,
+              private commentsToCommentTableModel: CommentsToCommentTableModelPipe,
+              protected statusMessageService: StatusMessageService,
+              protected commentService: FeedbackResponseCommentService,
+              protected commentToCommentRowModel: CommentToCommentRowModelPipe) {
```

Comment:
I think services can be all `private` here, since there is no more class extends from `InstructorSessionResultPageComponent `.

https://stackoverflow.com/a/48571668/9421552

Reply:
I can't make these private, only protected or no access modifier, otherwise I get `error TS6138: Property 'commentToCommentRowModel' is declared but its value is never read.` when running the front end test suite.

Also, I think the link you shared explains the difference between having an access modifier vs not having one, not about the difference between levels of access modifiers. So it matches up here since the distinction is between protected vs no access modifier and not protected vs private (if that makes sense).

Reply:
My mistake. I think no access modifier is right here to inherit from base class. 

https://stackoverflow.com/a/49520067/9421552